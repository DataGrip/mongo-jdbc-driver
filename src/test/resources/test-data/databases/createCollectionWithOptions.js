// before
// command
db.createCollection('col', {capped: true, size: 10});
// command
db.createCollection('col', {capped: true, size: 10, unknown: "a"});
// clear
db.col.drop();
