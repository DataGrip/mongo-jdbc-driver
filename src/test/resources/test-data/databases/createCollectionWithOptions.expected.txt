No result
com.mongodb.mongosh.result.CommandException: unrecognized create collection options field: unknown